src/conn/conn_dhandle.c

```diff
@@ -97,7 +95,9 @@ __wt_conn_dhandle_find(
 	conn = S2C(session);
 
 	/* We must be holding the handle list lock at a higher level. */
-	WT_ASSERT(session, F_ISSET(session, WT_SESSION_LOCKED_HANDLE_LIST));
+	WT_ASSERT(session,
+	    F_ISSET(session, WT_SESSION_LOCKED_READ_HANDLE_LIST |
```

Comment:
Is it worth creating a synonym of ``WT_SESSION_LOCKED_READ_HANDLE_LIST | WT_SESSION_LOCKED_WRITE_HANDLE_LIST`` as ``WT_SESSION_LOCKED_HANDLE_LIST``?